packages/cli/lib/gh.js

```diff
@@ -0,0 +1,99 @@
+import {
+  createStaticWorkspaceGHAction,
+  createDynamicWorkspaceGHAction
+} from 'create-platformatic'
+
+import parseArgs from 'minimist'
+import { access } from 'fs/promises'
+import pino from 'pino'
+import pretty from 'pino-pretty'
+
+export const createGHAction = async (logger, workspaceId, env, config, buildTS, type, projectDir = process.cwd()) => {
+  if (type === 'static') {
+    await createStaticWorkspaceGHAction(logger, workspaceId, env, config, projectDir, buildTS)
+    return
+  }
+  await createDynamicWorkspaceGHAction(logger, workspaceId, env, config, projectDir, buildTS)
+}
```

Comment:
I think this might be best exposed as `create-platformatic --actions` as part of `create-platformatic` project.

I would also add a `platformatic create` command to use `create-platformatic` with.

None of this should happen in this PR, so happy to land this as-is and then iterate in the coming weeks. In case, open an issue.